app/javascript/components/pages/Home.js

```diff
@@ -18,6 +19,36 @@ export class Home extends Component {
 							</Button>
 						</div>
 					</div>
+					<div className='home-content-container'>
+						<div className='home-content-text'>
+							<h3>
+								We are study up and this is just a for example and hopefully it
+								looks good lmao
+							</h3>
+						</div>
+						<div className='icon-container'>
+							<div className='icon-position1'>
+								<GiBookshelf size={35} className='icon-styling' />
+								<h1>
+									Step 1:
```

Comment:
It is best practice to only have one `h1` in the project. You can style the headers to look the same but the h1 has meaning for accessibility and SEO